Sources/IO/IOStream.swift

```diff
@@ -57,6 +57,8 @@ open class IOStream: NSObject {
         }
 
         /// NetStream has been created.
+        case uninitialized
```

Comment:
With the increase of "uninitialized," the number of states in readyState has increased, making it difficult to foresee and maintain. I also think "closed = uninitialized" is synonymous.

Reply:
If you modify createStream as follows, it ensures that the call to createStream will occur after the releaseStream command is received.
```
 func createStream(_ stream: RTMPStream) {
        let releaseStreamResponder = RTMPResponder(result: { data -> Void in
                let responder = RTMPResponder(result: { data -> Void in
            guard let id = data[0] as? Double else {
                return
            }
            stream.id = UInt32(id)
            stream.readyState = .open
           })
           call("createStream", responder: responder)
        })
        call(“releaseStream”, responder: releaseStreamResponder, arguments: stream.key)
    }
```
In modern times, it cannot be simply assumed whether various server-side implementations are correctly implemented or not. I think it would be good to call it like this.
```
func createStream(_ stream: RTMPStream) {
    // Specify an empty responder to ignore the result of releaseStream.
    let releaseStreamResponder = RTMPResponder(result: { data -> Void in })
    call(“releaseStream”, responder: releaseStreamResponder, arguments: stream.key)
    call(“createStream”, responder: …)
}
```

Reply:
I've updated the PR with optional releaseStream call without a responder.